docs/topic-guides/tls.txt

```diff
@@ -0,0 +1,68 @@
+.. _tls:
+
+##########################
+ Transport Layer Security
+##########################
+
+**Transport layer security** (TLS) is a protocol for applying encryption
+to network communications. It prevents the data being transmitted from
+being modified or read while it is in transit. Determined can be set up
+to use TLS for all inbound connections to the master. That means that
+all CLI and WebUI connections will be secured by TLS, as well as
+connections from agents and tasks to the master. Communications within
+the cluster that occur as part of :ref:`multi-machine training
+<multi-gpu-training>` are not protected by TLS, nor are proxied
+connections from the master to a TensorBoard or notebook.
+
+**************************
+ Determined Configuration
```

Comment:
I'd change this section title -- it's not about "Configuring Determined", it is about configuring TLS in Determined.